lib/http-bridge/test/integration/Test/Integration/Scenario/CLISpec.hs

```diff
@@ -119,6 +119,14 @@ specWithCluster = do
         err `shouldBe` "Ok.\n"
         out `shouldNotContain` "CLI Wallet"
         c `shouldBe` ExitSuccess
+
+    it "CLI - Can list addresses" $ \ctx -> do
+        walId <- createWallet ctx "CLI Wallet" mnemonics15
```

Comment:
why don't we also generate `mnemonics` here?

Reply:
Hmmm. Good question. No particular reasons that I can think of.